CMakeLists.txt

```diff
@@ -729,15 +731,14 @@ if(WIN32) # see options at: https://cmake.org/cmake/help/latest/cpack_gen/nsis.h
     # Extra install commands
     # Restores permissions on the install directory
     # Migrates config files from the root into the new config folder
-    # Sets permissions on the config folder so that we can write in it
     # Install service
     SET(CPACK_NSIS_EXTRA_INSTALL_COMMANDS
             "${CPACK_NSIS_EXTRA_INSTALL_COMMANDS}
         IfSilent +2 0
         ExecShell 'open' 'https://sunshinestream.readthedocs.io/'
         nsExec::ExecToLog 'icacls \\\"$INSTDIR\\\" /reset'
+        nsExec::ExecToLog 'icacls \\\"$INSTDIR\\\\config\\\\credentials\\\" /deny Users:\\\(R\\\)'
```

Comment:
It doesn't look like this does what we want because of inherited ACLs.

In my testing, this did the trick:
```
nsExec::ExecToLog 'icacls \\\"$INSTDIR\\\\config\\\\credentials\\\" /inheritance:r'
nsExec::ExecToLog 'icacls \\\"$INSTDIR\\\\config\\\\credentials\\\" /grant:r Administrators:\\\(OI\\\)\\\(CI\\\)\\\(F\\\)'
```

The first `icacls` invocation will disable inheritance, which strips all ACLs off and makes it accessible by nobody. The second one adds back access for the Administrators group.

Reply:
That's odd, not really sure why it worked on my machine then. When I tested it, it correctly denied permissions

Reply:
Yeah, it just leaves a bunch of stuff around that makes me uneasy like `APPLICATION PACKAGE AUTHORITY\ALL APPLICATION PACKAGES` and `APPLICATION PACKAGE AUTHORITY\ALL RESTRICTED APPLICATION PACKAGES`, so I think UWP apps may still have access to it.